apps/system-e2e/src/playwright.config.ts

```diff
@@ -15,10 +11,6 @@ const config: PlaywrightTestConfig = {
   /* Maximum time one test can run for. */
   timeout: 90 * 1000,
   expect: {
-    /**
```

Comment:
I am thinking this file is all about the configuration so it might be a good idea to leave the comments and commented out sections around for people to easily disable/enable them locally instead of needing to dig through the documentation for those.